documentation/sphinx/source/configuration.rst

```diff
@@ -387,6 +387,8 @@ FoundationDB will never use processes on the same machine for the replication of
 
     FoundationDB replicates data to three machines, and at least three available machines are required to make progress. This is the recommended mode for a cluster of five or more machines in a single datacenter.
 
+    .. note:: When running in cloud environments with managed disks that are already replicated and persistent, ``double`` replication may still be considered for 5+ machine clusters.  This will result in a lower total read throughput, but offers a reasonable tradeoff for cost.
```

Comment:
```suggestion
    .. note:: When running in cloud environments with managed disks that are already replicated and persistent, ``double`` replication may still be considered for 5+ machine clusters.  This will result in lower fault tolerance for planned or unplanned failures and lower total read throughput, but offers a reasonable tradeoff for cost.
```

Does this sound okay?

Reply:
I think that sounds pretty good. If you wanted, you could say "lower availability fault tolerance" to try to distinguish from durability, but I'll let you decide if that's a positive addition.

Reply:
I'm unable to commit the above change, but it sounds good to me.

Reply:
@alexmiller-apple If you can commit your suggestion, I'll merge this PR.

Reply:
I'm going to go ahead and merge this, then I'll make the proposed change in another PR.